src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/service/DigitalCaseFileServiceTest.java

```diff
@@ -154,4 +155,11 @@ void getReplyToReferralDCFLink_shouldReturnEmpty() {
         assertEquals(expected, actual);
     }
 
+    @Test
+    void shouldNotThrowNullPointerIfNoFileOrDocs() {
+        caseData.setDigitalCaseFile(null);
+        caseData.setDocumentCollection(null);
```

Comment:
do we need a test for the  scenario if file or doc actually exists?

Reply:
or maybe that test already exists?

Reply:
Yeah all the tests around docs existing or the DCF existing are before this test